sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpProducer.cs

```diff
@@ -222,9 +222,11 @@ internal class AmqpProducer : TransportProducer
         ///   Closes the connection to the transport producer instance.
         /// </summary>
         ///
+        /// <param name="isAsync"><c>true</c> if the method will be executed asynchronously; otherwise, false.</param>
         /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
         ///
-        public override async Task CloseAsync(CancellationToken cancellationToken)
+        public override async Task CloseAsync(bool isAsync,
```

Comment:
This definitely shouldn't be getting flagged by the analyzer, as its is an internal type and intentionally does not offer a synchronous surface.    What's the actual message that we're getting hit with and do you know what change caused it?